packages/lib-classifier/src/store/RootStore.js

```diff
@@ -23,6 +23,21 @@ const RootStore = types
     userProjectPreferences: types.optional(UserProjectPreferencesStore, UserProjectPreferencesStore.create())
   })
 
+  .volatile(self => {
```

Comment:
Is there a reason to set this in the root store rather than in the Subject store?

Reply:
Not really. I couldn’t decide whether component props should be stored in the root store or distributed through the store code. Putting it in the subject store makes sense to me too.

Reply:
If I move it, I'm not sure how to access `subject.setOnToggleFavourite` from the Classifier component. Would the callback have to be set each time a new subject is instantiated? I'm going to leave this as it is for the moment.

Maybe the callback should be stored on the store that holds the subject queue?